internal/cluster/common.go

```diff
@@ -66,6 +66,25 @@ func updateClusterStatus(log logrus.FieldLogger, db *gorm.DB, clusterId strfmt.U
 	return cluster, nil
 }
 
+func updateClusterProgress(log logrus.FieldLogger, db *gorm.DB, clusterId strfmt.UUID, srcStatus string,
+	progress string, extra ...interface{}) (*common.Cluster, error) {
+	var cluster *common.Cluster
+	var err error
+
+	extra = append(append(make([]interface{}, 0), "progress_progress_info", progress), extra...)
+
+	now := strfmt.DateTime(time.Now())
+	extra = append(extra, "progress_progress_updated_at", now)
+
+	if cluster, err = UpdateCluster(log, db, clusterId, srcStatus, extra...); err != nil {
```

Comment:
why not just return it? it looks like the same output no? i think that there should be an error log in the api level and not here.

Reply:
yup, done. 
see my reply to the comment on `inventory.go`